python/lsst/ts/scheduler/config_schema.py

```diff
@@ -116,440 +129,446 @@ class inside the module.
+                type: number
+                default: 3.5
+                description: Deceleration in altitude (degrees/sec^2).
+              azimuth_maxspeed:
+                type: number
+                default: 7.0
+                description: Maximum speed in azimuth (degrees/sec).
+              azimuth_accel:
+                type: number
+                default: 7.0
+                description: Accelaration in azimuth (degrees/sec^2).
+              azimuth_decel:
+                type: number
+                default: 7.0
+                description: Deceleration in azimuth (degrees/sec^2).
+              settle_time:
+                type: number
+                default: 3.0
+                description: Settle time.
+            additionalProperties: false
+          dome:
+            type: object
+            properties:
+              altitude_maxspeed:
+                default: 1.75
+                type: number
+                description: Dome maximum speed in elevation (degrees/s).
+              altitude_accel:
+                default: 0.875
+                type: number
+                description: Dome acceleration in elevation (degrees/s^2).
+              altitude_decel:
+                default: 0.875
+                type: number
+                description: Dome deceleration in elevation (degrees/s^2).
+              altitude_freerange:
+                default: 0.
+                type: number
+                description: >-
+                  Dome slit free-range in elevation (degrees). Specify how much
+                  the dome can move in elevation without blocking the telescope
+                  FoV.
+              azimuth_maxspeed:
+                default: 1.5
+                type: number
+                description: Dome maximum speed in azimuth (degrees/s).
+              azimuth_accel:
+                default: 0.75
+                type: number
+                description: Dome acceleration in azimuth (degrees/s^2).
+              azimuth_decel:
+                default: 0.75
+                type: number
+                description: Dome deceleration in azimuth (degrees/s^2).
+              azimuth_freerange:
+                default: 4.0
+                type: number
+                description: >-
+                  Dome slit free-range in azimuth (degrees). Specify how much the
+                  dome can move in azimuth without blocking the telescope FoV.
+              settle_time:
+                default: 1.0
+                type: number
+                description: >-
+                  Dome axis settle time (seconds). This is only applied if
+                  free_range is zero on the axis.
+            additionalProperties: false
+          rotator:
+            type: object
+            properties:
+              minpos:
+                default: -90.0
+                type: number
+                description: Minimum rotator position (degree).
```

Comment:
Here and in a few other places you use `degree` while in other places you use `degrees`. I think `degrees` is preferred for angles so please check and make sure to use `degrees` everywhere.